settings.gradle

```diff
@@ -1,15 +1,13 @@
+
 pluginManagement {
-    repositories {
-        gradlePluginPortal()
-        maven {
-            name = "NeoForge"
-            url = "https://maven.neoforged.net/releases/"
-        }
+    plugins {
+        //https://projects.neoforged.net/neoforged/moddevgradle
+        id('net.neoforged.moddev') version('2.0.75')
```

Comment:
Why when porting projecte did you define this in the normal plugins block, but here you are doing it in the settings gradle file?